src/components/PeriodicReports/PeriodicReportsTable.tsx

```diff
@@ -55,6 +55,8 @@ export const PeriodicReportsTableInContext = ({
       title: 'Period',
       defaultSort: 'desc',
       render: ({ report }) => <ReportLabel report={report} />,
+      customSort: (row1, row2) =>
+        row1.period.toMillis() - row2.period.toMillis(),
```

Comment:
I don't think this is necessary..

Reply:
Because of the custom render method which returns react element instead of raw string, the table wasnt able to be sorted by this column. That's why I had to add a new custom sort.

Reply:
Right of course 👍